package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eeacms/volto-eea-kitkat",
-  "version": "27.0.2",
+  "version": "27.0.3",
```

Comment:
@nileshgulia1 You should have at least 27.1.0 as volto-block-toc has a major version

Reply:
Done in [d8bde26](https://github.com/eea/volto-eea-kitkat/pull/201/commits/d8bde268c209e6d2a6095a2d852ce2295c20a11d)